audio-guestbook.ino

```diff
@@ -27,16 +34,46 @@
 #include <SPI.h>
 #include <SD.h>
 #include <TimeLib.h>
-#include <MTP_Teensy.h>
+#include <MTP_Teensy.h>  // this library has to be downloaded separately (https://github.com/KurtE/MTP_Teensy)
+// unzip the downloaded file and its content into your local Arduino folder (on my computer, the local Arduino folder is: "C:/Users/DD4WH/Documents/Arduino/libraries/")
+
+/***************************************************************************************************************************************************/
+/**USER CONFIGURATION ******************************************************************************************************************************/
+/**COMMENT / UNCOMMENT ACCORDING TO YOUR HARDWARE **************************************************************************************************/
+/***************************************************************************************************************************************************/
+
+// comment this out, if your handheld OPENS the contact on lift
+// use a digital voltmeter to find out
+#define HANDHELD_CLOSES_ON_LIFT
+
+// comment this out, if you want to record your greeting message with an external recorder
+// leave as-is if you want to have the telephone automatically switch to recording the greeting message, in case there is no "greeting.wav" on the SD card 
+#define AUTO_GREETING_MESSAGE
+
+// comment out, if you use a Teensy 4.0 (and thus the SD card slot on the audio board)
+// if you leave this as-is, you have to use the built-in SD card slot on the Teensy 4.1, NOT the SD card slot on the audio board 
+#define TEENSY_41
+
+/***************************************************************************************************************************************************/
+/**END OF USER CONFIGURATION ***********************************************************************************************************************/
+/***************************************************************************************************************************************************/
+/***************************************************************************************************************************************************/
 
-// DEFINES
 // Define pins used by Teensy Audio Shield
+#ifdef TEENSY_41
+#define SDCARD_CS_PIN    BUILTIN_SDCARD

```

Comment:
if I use a teensy 4.1 with the same sd card I get a very "jumpy" recording. it skips audio pieces every couple of seconds. if is comment that line and put the sd card into the audio shield it works fine. can you help me to understand what might happen here?